### PR TITLE
Feature/navigation

### PR DESCRIPTION
**Pull request by jalbe0076**
## What I did:
- Installed React Router dependencies
- Created a navigation bar and implemented Router to change the URL endpoints
- Created component for the URL endpoints including a component in case the user manually enters a non existing endpoint
- Styling is minimal but responsive

## Testing Notes/Extras: 
- No cypress testing integrated
- When you pull down the branch to test locally, run `npm i` to install React Router dependencies
- To confirm the sad path is working, manually enter a random endpoint that does not exist, the picture of a broken vinyl record should appear

(Include edge cases/potential sad paths to look out for)

Did this break anything?

- [X]  No
- [ ]  Yes

Type of change

- [X]  New feature (non-breaking change which adds functionality)
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [ ]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Skip all the other stuff and briefly explain the fix.

Checklist:

- [ ]  If this code needs to be tested, all tests are passing
- [X]  The code runs locally
- [ ]  There are comments where clarification/ organization is needed
- [X]  The code is DRY. If it's not, I tried my best
- [X]  I reviewed my code before pushing

What's next?
- Testing with cypress and discuss styling. Styling will need to change, the active Navlink was made red just to make it obvious.

### CODE REVIEW

**Review by jmartin777 — APPROVED** [review]
Great job Jason implementing React Router and creating the navigation components . Beginning the Cypress testing  and minimal responsiveness implementation is making me feel good about the ticket work flows we are setting as a group.

**Comment by alivaditis** [comment]
Great job, the broken record with the non-existent path is awesome.  Good job with routes and responsiveness.

**Review by alivaditis — APPROVED** [review]
Great job with the nav tabs

File: src/components/Nav/Nav.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  
+  .navigation-tabs {
+    grid-column: 1 / 3;
+    grid-row: 2 / 3;
+    justify-self: flex-start;
+    text-align: center;
+  }
+}
+
+@media (max-width: 380px) {
```
Comment (mbenfowler) [inline src/components/Nav/Nav.css:49]: great job with the responsiveness!

**Review by mbenfowler — APPROVED** [review]

**Pull request merged**